Jenkinsfile

```diff
@@ -55,31 +55,27 @@ pipeline {
       }
       steps {
         withMaven(jdk: 'jdk-8-latest', maven: 'maven-3.2-latest', mavenSettingsConfig: 'maven-nexus-settings') {
-          sh '''
-            mvn --version
-            java -version
-          '''
           nodejs('nodejs-14.6.0'){
-            sh '''
-              node -v
-              npm version
-            '''
              configFileProvider([configFile(fileId: 'maven-nexus-settings', variable: 'MAVEN_SETTINGS_XML')]) {
                sh """
                  mvn -s \$MAVEN_SETTINGS_XML clean install source:jar -Pdistro,distro-ce,distro-wildfly,distro-webjar -DskipTests -Dmaven.repo.local=\$(pwd)/.m2 com.mycila:license-maven-plugin:check -B
                """
              }
           }
-
-          archiveArtifacts artifacts: '.m2/org/camunda/**/*-SNAPSHOT/**', excludes: '**/*.zip,**/*.tar.gz', followSymlinks: false
-
+    
+          archiveArtifacts artifacts: '.m2/org/camunda/**/*-SNAPSHOT/**/*.jar,.m2/org/camunda/**/*-SNAPSHOT/**/*.pom,.m2/org/camunda/**/*-SNAPSHOT/**/*.xml,.m2/org/camunda/**/*-SNAPSHOT/**/*.txt,.m2/org/camunda/**/*-SNAPSHOT/**/camunda-webapp*frontend-sources.zip', followSymlinks: false
+          archiveArtifacts artifacts: '.m2/org/camunda/**/camunda-webapp*frontend-sources.zip,.m2/org/camunda/**/license-book*.zip,.m2/org/camunda/**/camunda-webapp*.war,.m2/org/camunda/**/camunda-engine-rest*.war,.m2/org/camunda/**/camunda-example-invoice*.war,.m2/org/camunda/**/camunda-h2-webapp*.war,.m2/org/camunda/**/*-SNAPSHOT/**/*.tar.gz,.m2/org/camunda/**/*-SNAPSHOT/**/camunda-jboss-modules*.zip', followSymlinks: false
```

Comment:
Let's add code comments to these two lines, so that it's clear what patterns should be added to each.
We should also be consistent with the `-SNAPSHOT` pattern, and use it in both places or in none of them.
Also, the `camunda-webapp*frontend-sources.zip` pattern is duplicated in both lines.